### DISCUSSION TITLE
Poor state of `podman` in Debian 11

### DISCUSSION THREAD

**Discussion by abitrolly** [containers/podman#14134] [Q&A]
Debian 11 (`bullseye` stable) comes with `podman` 3.0.1 https://packages.debian.org/search?suite=bullseye&keywords=podman that 

1. Doesn't work with `docker-compose` (https://github.com/containers/podman/discussions/13896#discussioncomment-2643783)
2. Mount volumes as `root`/`root` making them unwritable (https://github.com/Angatar/mailman2/issues/7#issuecomment-1118889466)

I've spent more than a week troubleshooting all these issues. Why leave the "stable" in this poor state?

**Comment by vrothberg** [comment]
Thanks for reaching out, @abitrolly.  Ultimately, that's more a Debian question than a Podman one.

Did you consider opening a Bugzilla against Debian?

**Reply by abitrolly** [reply]
Cross-posting each an every critical issue that is already fixed in `podman` to https://bugs.debian.org/cgi-bin/pkgreport.cgi?pkg=podman is not what I planned to do. Otherwise I wouldn't open this issue.

**Comment by andrewshadura** [comment]
Indeed, that’s a question for https://bugs.debian.org/podman.

**Comment by andrewshadura** [comment]
> I've spent more than a week troubleshooting all these issues. Why leave the "stable" in this poor state?

This is a strange question — podman *was* in this state when bullseye froze. You can’t update podman to something newer in a release, that’s a distro-wide policy. If you want newer versions of packages in stabe, use backports (although podman currently isn’t there).

**Comment by ne20002** [comment]
What do we need to do to get an updated version of Podman into stable-backports? 
On tracker.debian.org/podman I see 3.4.7 in testing. What's preventing this from being accepted into backports?

Also: if we won't get a backports version, what's the best way to get an updated version on Debian bullseye? Using debian testing repository? Using a suse repository?

**Reply by andrewshadura** [reply]
> What's preventing this from being accepted into backports?

There needs to be a person to do the work to backport it. Backports don’t happen automatically.
